conda/cli/main_config.py

```diff
@@ -610,24 +703,52 @@ def execute_config(args, parser):
 
             stdout_write("\n".join(format_dict(d)))
         context.validate_configuration()
+        context.plugins.validate_configuration()
         return
 
     if args.describe is not None:
         if args.describe:
-            paramater_names = args.describe
+            provided_parameters = tuple(
+                name for name in args.describe if not name.startswith("plugins.")
+            )
+            provided_plugin_parameters = tuple(
+                name.replace("plugins.", "")
+                for name in args.describe
+                if name.startswith("plugins.")
+            )
             all_names = context.list_parameters()
-            not_params = set(paramater_names) - set(all_names)
-            if not_params:
+            all_plugin_names = context.plugins.list_parameters()
+            not_params = set(provided_parameters) - set(all_names)
+            not_plugin_params = set(provided_plugin_parameters) - set(all_plugin_names)
+
+            if not_params or not_plugin_params:
                 from ..common.io import dashlist
                 from ..exceptions import ArgumentError
 
+                error_params = not_params | not_plugin_params
                 raise ArgumentError(
-                    f"Invalid configuration parameters: {dashlist(not_params)}"
+                    f"Invalid configuration parameters: {dashlist(error_params)}"
                 )
```

Comment:
Isn't this a near duplicate of the above? Can we use a helper function instead?

Reply:
Will add a `_validate_provided_parameters` function.